packages/compass-import-export/test/docs/all-bson-types.js

```diff
@@ -10,47 +10,51 @@ import {
   MaxKey,
   MinKey,
   ObjectId,
-  Symbol as BSONSymbol,
+  //Symbol as BSONSymbol,
```

Comment:
```suggestion
```
Might be nicer to remove the deprecated or leave a comment as to why it's commented. (Same with where `Symbol/BSONSymbol` is used below)